cluster/examples/kubernetes/ceph/create-external-cluster-resources.py

```diff
@@ -577,7 +600,7 @@ def _gen_output_map(self):
             self.out_map['CSI_CEPHFS_PROVISIONER_SECRET'] = self.create_cephCSIKeyring_cephFSProvisioner()
         self.out_map['RGW_ENDPOINT'] = self._arg_parser.rgw_endpoint
         self.out_map['MONITORING_ENDPOINT'], \
-            self.out_map['MONITORING_ENDPOINT_PORT'] = self.get_active_ceph_mgr()
+            self.out_map['MONITORING_ENDPOINT_PORT'] = self.get_active_and_standby_mgrs()
```

Comment:
wrong indent?

Reply:
Not really. If you see the above line, we have a back-slash (\) , which means a continuation of the line. So we are continuing from 602 line.